Resources/Prototypes/_Goobstation/Entities/Objects/Specific/Medical/hypospray.yml

```
@@ -53,28 +53,114 @@
+  description: Contains 7u of bicaridine and 3u of tranexamic acid, used in a cartridge autoinjector.
+  id: CartridgeBicaridine
+  components:
+  - type: Sprite
+    sprite: _Goobstation/Objects/Specific/Medical/cartridge.rsi
+    state: cartridge-bicaridine
+  - type: SolutionCartridge
+    targetSolution: hypospray
+    solution:
+      maxVol: 10
+      reagents:
+      - ReagentId: Bicaridine
```

yamllint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)